index.html

```diff
@@ -1,9 +1,132 @@
+<html>
+
+<head>
+    <link rel="stylesheet" href="style.css" />
+</head>
+
+<body>
+    <header class="header">
+        <div>
+            <div>
+                <img class="blackHeart" src="images/black-heart.png" alt="black heart" />
+                <span class="nifty"> Nifty Penguin Magic </span>
+            </div>
+            <nav class="navBar">
+                <ul class="nav-bar-ul">
+                    <li><a class="nav-bar-item" href="#npmEnterprise"> npm Enterprise</a></li>
+                    <li><a class="nav-bar-item" href="#Products"> Products</a></li>
+                    <li><a class="nav-bar-item" href="#Solutions"> Solutions</a></li>
+                    <li><a class="nav-bar-item" href="#Resources"> Resources</a></li>
+                    <li><a class="nav-bar-item" href="#Docs"> Docs</a></li>
+                    <li><a class="nav-bar-item" href="#Support"> Support</a></li>
+                    <li><a class="nav-bar-item" href="#Search"> Search</a></li>
+                    <li><a class="nav-bar-item" href="#Join"> Join</a></li>
+                    <li><a class="nav-bar-item" href="#LogIn"> Log In</a></li>
+                </ul>
+            </nav>
+        </div>
+
+        <div class="npmrow">
+            <img class="npmLogo" src="images/npm-logo.png" alt="npm logo" />
+
+            <!-- 
+            <form>
+                <input type="text" class="searchbar">
+            </form>
+            <!-- add logo, search bar and buttons (or links and style them as buttons later)
+            <img class="magnifyingGlass" src="images/magnifying-glass.png" alt="magnifying glass" />
+        -->
+
+
+
+            <div class="w50">
+                <div class="flexbox">
+                    <input type="text" id="searchtext" placeholder="Search packages" />
+                    <input type="button" class="searchbtn" placeholder="Search"/>
+                </div>
+            </div>
+
+            <div class="slot-form-container">
+
+                <button type="button">Join</button>
+                <button type="button">Log in</button>
+            </div>
+
+        </div>
+    </header>
+
+    <section id="build">
+
+        <article class="container">
+            <h1 class="buildtitle"> Build amazing things </h1>
+            <p class="text">Essential JavaScript development tools that help you <br> go to market faster and build
+                powerful
+                applications <br> using modern open source code.</p>
+
+            <!--  <button class="btn-plans" type="button"> See plans </button>
+            <button class="btn-join" type="button"> Join for free </button> -->
+
+            <a class="btn-plans" href="#" alt="see plans"> See plans </a>
+            <a class="btn-join" href="#" alt="join for free"> Join for free </a>
+
+        </article>
+    </section>
+
+    <img class="triangleExagon" src="images/triangle-hexagon.svg" alt="triangle hexagon shape" />
+
+
+    <section class="">
```

Comment:
Nice use of section tags. What about using a class or id that describes the section content?